### ISSUE TITLE
`terraform import` fails for `exoscale_domain` with "an object with 7 attributes is required (9 given)"

### ISSUE THREAD

**Issue by huerlisi** [exoscale/terraform-provider-exoscale#238]
**Versions:**
* terraform v1.3.7
* registry.terraform.io/exoscale/exoscale v0.43.0
* exo 1.64.0 e906dc80 (egoscale 0.96.0)

**Problem:**
I'm trying to import an `exoscale_domain` like this: `terraform import exoscale_domain.xyz_org 11111111-b648-474a-0000-0000000f344a` and get the following error:

```
exoscale_domain.xyz_org: Importing from ID "11111111-b648-474a-0000-0000000f344a"...
╷
│ Error: an object with 7 attributes is required (9 given)
```

**Notes:**
Importing a single record works fine: `terraform import exoscale_domain_record.x_xyz_org 11111111-b648-474a-0000-0000012f2f4c`:

```
exoscale_domain_record.x_denteo_org: Importing from ID "11111111-b648-474a-0000-0000012f2f4c"...
exoscale_domain_record.x_denteo_org: Import prepared!
  Prepared exoscale_domain_record for import
exoscale_domain_record.x_denteo_org: Refreshing state... [id=11111111-b648-474a-0000-0000012f2f4c]

Import successful!

The resources that were imported are shown above. These resources are now in
your Terraform state and will henceforth be managed by Terraform.
```

**Comment by kobajagi** [comment]
Hi. Thank you for letting us know about the issue. Currently import of `exoscale_domain` and `exoscale_security_group` resources don't work with terraform 1.3.x versions. Issue will be resolved when PR #239 is merged. As a workaround terraform versions 1.2.x may be used.

**Comment by kobajagi** [comment]
Relevant PR merged. Note that nested import (importing `exoscale_domain_record` as part of `exoscale_domain`) is no longer possible and you must import each resource individually.
